store/memory/store_test.go

```diff
@@ -0,0 +1,32 @@
+package memory_test
```

Comment:
Do you have any documentation around this? Why not use “package store” here?

Reply:
Could you elaborate on where you'd like to see more documentation?  Just wasn't sure what you were hoping to see.  I added a package doc but I'm also wondering if that would be more noticeable if I put it on the New method (someone might be more likely to see it in their IDE).

Reply:
sorry if I wasn't clear here; I prefer this package to be `store` and then you won't have to manually import the store package.

For example the `redis` package test file is not `redis_test` it's just `redis`:
https://github.com/harlow/kinesis-consumer/blob/master/store/redis/redis.go#L1-L5